web/themes/custom/votegov/src/sass/uswds-overrides/usa-in-page-nav.scss

```diff
@@ -22,12 +22,15 @@
     }
 
     &.usa-current {
-      @include vote-link-bg($color: $base-dark);
+      // @include vote-link-bg($color: $base-dark);
```

Comment:
i would take this line out... just have it as `@include vote-link-bg` and then make a comment above that after the fix to add back `($color: $base-dark)`